src/puripuly_heart/core/managed_openrouter_broker_client.py

```
@@ -174,6 +177,46 @@ async def issue_discord_managed_key(
+                openrouter_api_key=_require_text(payload, "openrouter_api_key"),
+                managed_credential_ref=_require_optional_text(payload, "managed_credential_ref"),
+                expires_at=_require_optional_text(payload, "expires_at"),
+                openrouter_user_id=normalize_managed_openrouter_user_identifier(
+                    payload.get("openrouter_user_id")
+                ),
+                referral_bonus_applied=_parse_referral_bonus_applied(payload),
+                referral_id=_parse_owned_referral_id(payload),
+                pass_status=_parse_talk_together_pass_status(payload),
+            )
+            logger.info("[QQAuth] assert_qq_credential: success, got openrouter_api_key")
+            return result
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)